api/module/ModuleRouter.go

```diff
@@ -18,6 +18,7 @@ func NewModuleRouterImpl(moduleRestHandler ModuleRestHandler) *ModuleRouterImpl
 
 func (impl ModuleRouterImpl) Init(configRouter *mux.Router) {
 	configRouter.Path("").HandlerFunc(impl.moduleRestHandler.GetModuleInfo).Queries("name", "{name}").Methods("GET")
```

Comment:
why do we need this ??

Reply:
Tested the api route, golang didn't take queries as optional, but I wanted to make it optional, that's why I had put it there